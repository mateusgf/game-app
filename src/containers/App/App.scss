.App {
  padding: 20px;

  @media (min-width: 992px) {
    margin: 50px auto 0 auto;
    max-width: 460px;
    box-shadow: 19px 16px 42px 3px rgba(0, 0, 0, 0.1);
    -webkit-box-shadow: 19px 16px 42px 3px rgba(0, 0, 0, 0.1);
    -moz-box-shadow: 19px 16px 42px 3px rgba(0, 0, 0, 0.1);
    border: 1px #efefef solid;
    height: 90vh;
  }
}

table,
th,
td {
  border: 1px solid black;
  border-collapse: collapse;
}
