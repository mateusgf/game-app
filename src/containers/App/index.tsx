import "./App.scss";

import PlayerProvider from "../../context/Player";
import GameProvider from "../../context/Game";

interface IAppProps {
  children?: React.ReactNode;
}

function App({ children }: IAppProps) {
  return (
    <PlayerProvider>
      <GameProvider>
        <div className="App" data-testid="app">{children}</div>
      </GameProvider>
    </PlayerProvider>
  );
}
export default App;
