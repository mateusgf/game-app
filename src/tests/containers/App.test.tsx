import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import App from "../../containers/App";


describe("App container", () => {
  test("match snapshot", () => {

    const { container } = render(<App/>);
    expect(container).toMatchSnapshot();
  });

});
