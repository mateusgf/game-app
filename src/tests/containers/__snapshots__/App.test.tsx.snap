// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App container match snapshot 1`] = `
<div>
  <div
    class="App"
    data-testid="app"
  />
</div>
`;
